airbyte-integrations/connectors/source-amplitude/source_amplitude/streams.py

```diff
@@ -206,3 +209,11 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
 
     def path(self, **kwargs) -> str:
         return f"{self.api_version}/export"
+
+    def get_error_handler(self) -> ErrorHandler:
+        error_mapping = DEFAULT_ERROR_MAPPING | {
+            400: ErrorResolution(response_action=ResponseAction.IGNORE, failure_type=FailureType.config_error),
+            404: ErrorResolution(response_action=ResponseAction.IGNORE, failure_type=FailureType.config_error),
```

Comment:
Maybe a dumb question, but do we ignore `400` and `404` errors, but not the `504`? Based on the above code, do a custom ignore in `read_records()` to skip over those types of http errors when we receive them.

My other question is, if we're doing the error catching and ignoring in the retriever now, do we still need the try/catch in `read_records()`? And if we get rid of it there, are we losing the extra log statements?

Reply:
Indeed as the exception raised from now on would probably be of type AirbyteTracedException

Reply:
Updated to no longer ignore 400/403 errors. 404 is still ignored. 504 raises a transient error.